pkgs/development/python-modules/shap/default.nix

```
@@ -35,29 +38,49 @@ buildPythonPackage rec {
     tqdm
     slicer
     numba
+    cloudpickle
   ];
 
   preCheck = ''
     export HOME=$TMPDIR
     # when importing the local copy the extension is not found
     rm -r shap
   '';
-  checkInputs = [ pytestCheckHook matplotlib nose ipython ];
+  checkInputs = [
+    pytestCheckHook
+    matplotlib
+    nose
+    ipython
+    pytest-mpl
+    pytest-cov
+  ];
   # Those tests access the network
-  disabledTests = [
-    "test_kernel_shap_with_a1a_sparse_zero_background"
-    "test_kernel_shap_with_a1a_sparse_nonzero_background"
-    "test_kernel_shap_with_high_dim_sparse"
-    "test_sklearn_random_forest_newsgroups"
-    "test_sum_match_random_forest"
-    "test_sum_match_extra_trees"
-    "test_single_row_random_forest"
-    "test_sum_match_gradient_boosting_classifier"
-    "test_single_row_gradient_boosting_classifier"
-    "test_HistGradientBoostingClassifier_proba"
-    "test_HistGradientBoostingClassifier_multidim"
-    "test_sum_match_gradient_boosting_regressor"
-    "test_single_row_gradient_boosting_regressor"
+  pytestFlagsArray = [
+    "--deselect" "tests/explainers/test_kernel.py::test_kernel_shap_with_a1a_sparse_zero_background"
```

why not use the old `disabledTests` paradigm?
```suggestion
  disabledTests = [
    "test_kernel_shap_with_a1a_sparse_zero_background"
```

---

for some reason these were not working anymore